services/integration/src/models/constitution/constitution.models.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const constitutionSchema = new Schema({
+  sponsor: {
+    type: String,
+    required: [true, 'sponsor obligatory field']
+  },
+  customer: String,
+
+  assignment: Boolean,
+
+  contract: Boolean,
+
+  businessCase: Boolean,
+
+  utterance: Boolean,
+
+  strategicView: {
+    type: String,
+    required: [true, 'strategic view obligatory field']
+  },
+  descriptionProject: {
```

Comment:
why didn't you change this attribute name?

Reply:
which attribute name?

Reply:
The comments are attached to the line above them. Im talking about the `descriptionProject` attribute name. I commented you this previously.